CMakeLists.txt

```diff
@@ -21,22 +21,6 @@ if(ITKPythonPackage_SUPERBUILD)
   option ( ITKPythonPackage_BUILD_PYTHON "Build ITK python module" ON )
   mark_as_advanced( ITKPythonPackage_BUILD_PYTHON )
 
-  # Set a default build type if none was specified
-  if(NOT CMAKE_BUILD_TYPE AND NOT CMAKE_CONFIGURATION_TYPES)
-    message(STATUS "Setting build type to 'MinSizeRel' as none was specified.")
-    set(CMAKE_BUILD_TYPE MinSizeRel CACHE STRING "Choose the type of build." FORCE)
-    mark_as_advanced(CMAKE_BUILD_TYPE)
-    # Set the possible values of build type for cmake-gui
-    set_property(CACHE CMAKE_BUILD_TYPE PROPERTY STRINGS "Debug"
-      "Release" "MinSizeRel" "RelWithDebInfo")
-  endif()
-  set(ep_common_cmake_cache_args)
-  if(NOT CMAKE_CONFIGURATION_TYPES)
-    list(APPEND ep_common_cmake_cache_args
-      -DCMAKE_BUILD_TYPE:STRING=${CMAKE_BUILD_TYPE}
```

Comment:
It seems that ITK will not be build with the expected build type anymore. Or is the value already propagated by ExternalProject ?